packages/generator-react-server/README.md

```diff
@@ -0,0 +1,42 @@
+# generator-react-server [![NPM version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Coverage percentage][coveralls-image]][coveralls-url]
+> A react-server yeoman generator
```

Comment:
I don't actually know if the badges are going to work out of the box, because of the monorepo stuff.  I do know that they won't work as instrumented because the redfin fork of this package doesn't exist yet.  I'd like to merge with the understanding that I'll have to iterate, then publish a 0.2.7